atecc608a_se.c

```diff
@@ -125,13 +125,13 @@ psa_status_t atecc608a_to_psa_error(ATCA_STATUS ret)
  * format for pubkeys is 0x04 + x + y. Always use a pubkey buffer in PSA
  * format, with enough space for the PSA format. To translate this buffer for
  * use with cryptoauthlib, use pubkey_for_driver(). To ensure the buffer is in
- * valid PSA format after cryptoauthlib operations, call pubkey_for_psa(). */
+ * valid PSA format after cryptoauthlib operations, call add_psa_pubkey_prefix(). */
 static uint8_t *pubkey_for_driver(uint8_t *data)
 {
     return &data[1];
 }
 
-static void pubkey_for_psa(uint8_t *data)
+static void add_psa_pubkey_prefix(uint8_t *data)
```

Comment:
Minor: “add” makes me think this appends something, or at least increases the length. But this actually prepends the prefix, and modifies an existing buffer without changing its size. This looks like a `set_psa_pubkey_prefix` to me.

Reply:
Done.